index.html

```diff
@@ -557,24 +566,66 @@ <h3>Terminology</h3>
-such as update a [=controller document=] or use a cryptographic key to generate
-a digital signature.
+            <p>
+An entity that is capable of performing an action with a specific resource,
+such as updating a [=controller document=] or generating a [=proof=] using a [=verification method=].
+            </p>
           </dd>
 
           <dt><dfn class="export">controller document</dfn></dt>
           <dd>
-A set of data that specifies one or more relationships between a
-[=controller=] and a set of data, such as a set of public cryptographic keys.
+            <p>
+A document that contains cryptographic material and service
+endpoints for the purposes of verifying proofs from, and interacting
+with, the [=controller=] of an identifier.
+            </p>
           </dd>
 
           <dt><dfn data-lt="subjects">subject</dfn></dt>
           <dd>
-The entity identified by the `id` property in a [=controller document=].
-Anything can be a subject: person, group, organization, physical thing, digital
-thing, logical thing, etc.
+            <p>
+An entity that is referred to by the `id` property in a [=controller document=]
+when it is used as a subject in other contexts, such as authentication or
+attestations, e.g., Verifiable Credentials. Anything can be a subject: 
+person, group, organization, physical thing, digital thing, logical thing, etc. 
+            </p>
+            <p class="note">
+It is expected that the subject referred to by the id of a controller document 
+be consistent over time, so that a set of credentials with a given identifer as 
+subject can be taken to be about the same entity. However, it *is* possible to 
+issue credentials with a subject identifier _without_ the subject even being 
+aware of it.
+
+It is expected that credential issuers will ask the subject for identifiers 
+and that those subjects would demonstrate proof of control before the issuer
+issues the credential. However, there are valid cases where issuers will 
+use identifiers that are not provably under the control of the subject. 
+
+However, even in cases
+where the subject of the ID does prove control, the interpretation of the subject
+remains contextual and potentially ambiguous.
+
+For example, the White House might issue a credential about the president of the United States 
+using did:example:abc as a subject identifier, saying in effect "The President 
+likes fried chicken." Then, the US National Parks Service might issue their own 
+credential about the president, using the same identifier, saying "The President authorizes
+the creation of new National Parks." 
+
+However, the White House meant "President Joe Biden", who is a physical person who 
+happened to be in office at the time the credential was issued. In contrast, 
+the park service meant "the current president, aka POTUS, whoever he is"; this second VC is not 
+about the Joe Biden as a human individual, it's about the role of President of
+the United States. 
```

Comment:
This is problematic. URLs are specified as always denoting the same entity. DIDs are URLs. `did:example:abc` should be described as either "President Joe Biden" or "the current POTUS".

Reply:
This is a problem with RDF, not with the example.

Reply:
I encourage you to log your issue with RDF soon, such that it might be taken up, or at least noted, by the currently active RDF-star WG.

Reply:
RDF knows about this issue. It is called HttpRange14.

As long as RDF retains a global context for all identifiers such that any use of that identifier necessarily refers to a subject other than the identifier itself, then it is impossible, within RDF to have a statement about the identifier using that identifier.

In the real world, we clarify by context which usage is appropriate. In one context we understand that we are making statements about the subject referred to by an identifier, in others, we understand we are talking about the identifier itself. RDF's treatment of identifiers makes this problematic.